src/custom.scss

```diff
@@ -21,12 +21,12 @@ body {
 .navbar-custom {
   min-height: 56px;
   background: $background; /* Old browsers */
-  background: -moz-linear-gradient(top,  $backgroundColor1 0%, $backgroundColor2 53%, $backgroundColor3 100%); /* FF3.6+ */
+  background: -moz-linear-gradient(top,  rgba(46,143,177,1) 0%, rgba(33,134,169,1) 53%, rgba(33,134,169,1) 100%); /* FF3.6+ */
```

Comment:
Why did you decide to switch back to numbers?

Reply:
it was not working on reach3.ezuce.com... The background color was different there

Reply:
How is it even possible?

Reply:
I will check later but on reach3 is used background: linear-gradient(top,#2e8fb1,#2186a9 53%,#0d7799); which is incorrect and on my localhost is used background: -moz-linear-gradient(top, #2e8fb1 0%, #2186a9 53%, #0d7799 100%); which is correct